src/javascript/app_2/Stores/Modules/Contract/Helpers/details.js

```diff
@@ -53,6 +52,6 @@ export const getDetailsExpiry = (store) => {
             [localize('Exit Spot')]     : end_spot ? addComma(end_spot) : '-',
             [localize('Exit Spot Time')]: end_spot_time ? toGMTFormat(+end_spot_time * 1000) : '-',
         }),
-        [localize('Payout')]: <Money amount={indicative_price} currency={currency} />,
+        [localize('Payout')]: <Money amount={indicative_price} currency={store.contract_info.currency} />,
```

Comment:
since we already got `contract_info` in line 38, it's better to get currency from there: `currency={contract_info.currency}`